### ISSUE TITLE
ModuleNotFoundError: No module named 'aragog'

### ISSUE THREAD

**Issue by stefsmeets** [FormingWorlds/PROTEUS#250]
Aragog is missing from the dependency list.

**Comment by stefsmeets** [comment]
After installing aragog, I get this error running the tests (python 3.12):

```
Traceback:
/usr/lib64/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/integration/test_integration_physical.py:14: in <module>
    from proteus.interior.aragog import read_ncdf as read_interior
src/proteus/interior/aragog.py:12: in <module>
    from aragog import Output, Solver
E   ImportError: cannot import name 'Output' from 'aragog' (/home/stef/python/proteus/.venv/lib/python3.12/site-packages/aragog/__init__.py)
```

**Comment by stefsmeets** [comment]
Seems like the version on pypi is very old: https://github.com/ExPlanetology/aragog/issues/24

**Comment by stefsmeets** [comment]
It seems to me that this breaks the proteus installation and tests if you want to use aragog. Proteus should add aragog to its dependencies and set the minimum required version. It would also help if aragog had a current release on pypi, otherwise this makes proteus impossible to install for non-technical users.

**Comment by nichollsh** [comment]
I have made a release on the aragog side of things, which has been uploaded to PyPI. 

We just need to update the PROTEUS pyproject file and fix this core_radius keyword error.

Once this issue is resolved we can address #245, #227,  #242, #240, etc.

**Comment by lsoucasse** [comment]
Hi @nichollsh, would you take care of this in a separate branch or in Stef or Emma current PR. It should be just adding aragog as a dependency in the pyproject.toml, remove aragog download in the workflow and delete the core_radius entry if I am correct.

**Comment by nichollsh** [comment]
Yeah sure! We have fixed the core_radius thing in Emma's escape branch, but the pyproject thing can be done in a new branch.
